README.md

```diff
@@ -69,6 +69,27 @@ hCaptcha
         Log.d("hCaptcha", "hCaptcha is now visible.");
     }
   });
+  .addOnCloseListener(new OnCloseListener() {
+    @Override
+    public void onClose() {
+        // Usefull for analytics purposes
+        Log.d("hCaptcha", "hCaptcha is closed");
+    }
+  })
+  .addOnExpiredListener(new OnExpiredListener() {
```

Comment:
This would not work since the hCaptcha JS API is removed after a successful verification and thus, cannot trigger this callback.

Reply:
Thanks for the note.
I understand how to reproduce the "Challenge Expired" case, but how (in theory) "Expired" can be reproduced?

Reply:
To reproduce "expired", you can try our [demo page](https://accounts.hcaptcha.com/demo), solve a captcha, wait some time and the successful verification will expire and the captcha will reset. This cannot happen in the Android flow as the webview will be cleaned up after a successful verification. We cannot guarantee this for the dialog flow.

Reply:
@DSergiu thanks for the explanation. I agree that `.addOnExpiredListener` doesn't make sense in this case, I will remove it

Reply:
Done

Reply:
@DSergiu the way I'd think of this happening on the Android SDK is to fire that listener N seconds after a valid token is issued, letting the app know it's expired.

Reply:
@e271828- can we get the N value from WebView somehow or should we hardcode it? BTW is this make any sense to the developers, I mean for which case they may need to know that token was expired?

Reply:
We should leave it configurable by developers via compile-time var, but default to 120 seconds.

Reply:
hmm, it will not relate to a 'real' token expiration (which probably exists inside hcaptcha web service), but more kind of 'artificial' mechanism for getting to know that some time passed after the token received.

I still trying to find/understand some real-world case... but ok.

Additional changes:
 - add `expirationTimeout` to HCaptchaConfig
 - implement some `removeOnExpiredListener` callback in case if the end-user in the app will navigate out of current activity (or implement own `Application.ActivityLifecycleCallbacks` to listen for this internally)

@DSergiu could you please check my proposed changes, maybe I missed something or it's overkill?

Reply:
@CAMOBAP since the expiration is (generally) fixed, just letting the app know that token is expired and must be refreshed is valuable. A listener/callback is a convenient mechanism.

Reply:
Done